Update export-pkg error description (#1624)
conan-io/docs: reference/commands/creator/export-pkg.rst

   and the ``self.copy(...)`` functions will copy matching files from the ``source_folder``
   **and** ``build_folder`` to the corresponding path in the Conan cache (working directory
   corresponds to the ``build_folder``).
 * If the arguments ``--package-folder``, ```--build-folder`` or ``--source-folder`` are

>    declared, but the path is incorrect, :command:`export-pkg` will raise a ``ConanException``.

Reviewer: I don't think the exception type is important here
```suggestion
   declared, but the path is incorrect, :command:`export-pkg` will raise an exception.
```
